src/CMakeLists.txt

```
@@ -27,5 +27,6 @@ target_compile_options(sirit PRIVATE ${SIRIT_CXX_FLAGS})
 
 target_include_directories(sirit
                            PUBLIC ../include
-                           PRIVATE . ${SPIRV-Headers_SOURCE_DIR}/include
-                           INTERFACE ${SPIRV-Headers_SOURCE_DIR}/include)
+                           PRIVATE .)
+
+target_link_libraries(sirit PUBLIC SPIRV-Headers)
```

```suggestion
target_link_libraries(sirit PUBLIC SPIRV-Headers::SPIRV-Headers)
```

Linking to targets containing `::` is preferred; citing [the docs](https://cmake.org/cmake/help/latest/command/target_link_libraries.html):

> Items containing `::`, such as `Foo::Bar`, are assumed to be [IMPORTED](https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#imported-targets) or [ALIAS](https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#alias-targets) library target names and will cause an error if no such target exists. See policy [CMP0028](https://cmake.org/cmake/help/latest/policy/CMP0028.html#policy:CMP0028).

Otherwise the build will fail later, at the compilation/linking phase instead of configuration.

---

Yes it's better, but I was trying to minimize code change with the original source :p